docs/design/index.md

```diff
@@ -77,24 +77,24 @@ summarization partially pre-aggregates your data, and can lead to big costs savi
 ## When should I use Druid?
 
 Druid is used by many companies of various sizes for many different use cases. Check out the
-[Powered by Apache Druid](/druid-powered) page
+[Powered by Apache Druid](/druid-powered) page.
 
 Druid is likely a good choice if your use case fits a few of the following descriptors:
 
 - Insert rates are very high, but updates are less common.
-- Most of your queries are aggregation and reporting queries ("group by" queries). You may also have searching and
+- Most of your queries are aggregation and reporting queries (group by queries). You may also have searching and
```

Comment:
Actually "group by" is the name of one of our query types. The [doc](https://druid.apache.org/docs/latest/querying/groupbyquery.html) mostly uses "groupBy". How about using the same notation here too?